components/docs-chef-io/content/automate/servicenow_integration.md

```diff
@@ -0,0 +1,221 @@
+
+## Chef Automate and ServiceNow Integration
+
+The integration between a Chef Automate server and a ServiceNow instance requires the following components:
+
+- Chef Automate Scoped Application
+- Chef Automate Server
+
+The Chef Automate application is a ServiceNow certified scoped application available from the ServiceNow store.  It integrates with the Chef Automate infrastructure and compliance functionality.
+
+The [Chef Automate](https://www.chef.io/automate/) provides a full suite of enterprise capabilities for workflow, node visibility, and compliance. The Chef Automate server sends HTTPS JSON data feeds to the Chef Automate application in a ServiceNow instance to create and update the _ServiceNow CMDB_ tables, the application client run and compliance report tables.
+
+{{< figure src="/images/automate/snow_integration_dataflow_diagram.png" alt="Data Flow Diagram">}}
+
+## Installation
+
+### Prerequisites
+
+- The ServiceNow instance must be publicly reachable on https port 443.
+- [Chef Automate Server](https://docs.chef.io/chef_automate.html).
+- ServiceNow plugin: System Import Sets `com.glide.system_import_set`, min version 1.0.0.
+- ServiceNow plugin: Configuration Management (CMDB) 1.1.
+- ServiceNow plugin: Configuration Management for Scoped Apps [com.snc.cmdb.scoped] 1.0.0.
+
+The installation personnel performing the task of installing and configuring the Chef Automate Integration app must be aware of the ServiceNow ecosystem.
+
+{{< note >}}
+
+You can obtain the **System Import Sets**, **CMDB 1.1**, and **Configuration Management for Scoped App** plugins by navigating to the **System Applications** > **All Available Applications** > **All** section in the ServiceNow application. Alternatively, you can also find them in the **Plugins** section.
+
+{{< figure src="/images/automate/snow_integration_plugins.png" alt="Plugins">}}
+
+{{< /note >}}
+
+### Installing Chef Automate application in ServiceNow
+
+The Chef Automate application exposes the REST API endpoint that facilitates the communication between Chef Automate and the ServiceNow instance.
+
+1. Visit the ServiceNow store at <https://store.servicenow.com>.
+2. Get the **Chef Automate Integration App** application.
+3. Navigate to the ServiceNow instance.
+4. Select **System Applications** > **All Available Applications** > **All** menu.
+5. Find the application using the filter criteria and search bar.
+   Note that you can search for the application by its Name or ID. If you cannot find an application, you may have to request it from the ServiceNow Store.
+6. Select **Install**.
+
+### Creating Application Users
```

Comment:
I am very confused by these sections. If I am setting up the Chef Automate ServiceNow integration, what actions do I need to take to set up the application, and what is the order in which I need to take them?

Reply:
Its intall, assigning user roles, and then config in either of the two ways. Made relevant changes. Hope now it clarifies your question. Please check and lemme know.